### PR TITLE
eyeddx

### PR DESCRIPTION
**Pull request by pgop88**
Differential diagnosis of eye complaints using rule of 2s.

### CODE REVIEW

File: cards/eyeddx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+This approach is not meant to be all-inclusive of ocular diagnoses but rather is meant to provide a framework one can use to guide the physical exam, such as while using the slit lamp, and to systematically include or eliminate certain diagnoses.
+
+## ANTERIOR EYE – utilize slit lamp exam
```
Comment (dsifford) [inline cards/eyeddx]: Change to title case `Anterior Eye: Utilize slit lamp exam`

File: cards/eyeddx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+![](image-1.png)
+
+## POSTERIOR EYE – use fundoscopy and ultrasound
```
Comment (dsifford) [inline cards/eyeddx]: Change to title case `Posterior Eye: Use fundoscopy and ultrasound`

File: cards/eyeddx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+![](image-2.png)
+
+## OTHER EYE DIAGNOSES
```
Comment (dsifford) [inline cards/eyeddx]: Change to title case

File: cards/eyeddx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+![](image-3.png)
+
+# REFERENCES
```
Comment (dsifford) [inline cards/eyeddx]: Change to title case

File: cards/eyeddx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Nawar, E.W., Niska, R.W., Xu, J. National Hospital Ambulatory Medical Care Survey: 2005 Emergency Department Summary: Advance Data from Vital and Health Statistics; No. 386. National Center for Health Statistics, Hyattsville, MD; 2007.
```
Comment (dsifford) [inline cards/eyeddx]: Should be an unordered list.

**Comment by dsifford** [comment]
Also, thanks so much for contributing @pgop88! Looking forward to more in the future! 😄

File: cards/eyeddx (hunk 4 above)
Comment (jvoros) [inline cards/eyeddx]: Should be H2 
```
## References
```
See the style guide: https://github.com/aliemteam/aliemcards/wiki/Markdown-Style-Guide

File: cards/eyeddx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+title: See the Unseen: Use the Rule of 2s to Never Miss Another Ocular Emergency
```
Comment (jvoros) [inline cards/eyeddx]: I would use a less editorial title. Something like "Rule of 2s for Ocular Emergencies"

Comment (pgop88) [inline cards/eyeddx]: Done!

**Comment by dsifford** [comment]
@pgop88 Thanks for fixing the items brought up by @jvoros. We still need you to also fix the things I mentioned before this can be merged.

Thanks 😄

**Comment by jvoros** [comment]
@pgop88 thank you so much for working through this! We set up this new system, and we're trying to figure out how it will work with the community as a whole. So thank you for being a 'guinea pig'.

I've added some screencasts to the Wiki tutorials.

Check out this one on Building a Card: https://github.com/aliemteam/aliemcards/wiki/How-to-Build-an-ALiEM-Card

And then I've updated the Adding a Card tutorial with video help as well: https://github.com/aliemteam/aliemcards/wiki/Tutorial:-Adding-a-Card

**Comment by pgop88** [comment]
@jvoros The videos you created are amazingly helpful and demystified the entire process! Had no idea it was really this simple to create a card. Can't believe I was struggling so much. 
Thank you again!!

By the way
@dsifford @jvoros just created a new card/folder and incorporated the changes you mentioned for this card!

**Comment by dsifford** [comment]
Thanks @pgop88. Closing this Pull Request in favor of #170

**Pull request closed without merging**